masterlist.yaml

```diff
@@ -712,3 +721,67 @@ plugins:
         <<: *dirtyPlugin
         itm: 4
         udr: 1
+  - name: 'AyleidCitadel.esp'
+    tag:
+      - C.Regions
+  - name: '3DNPC.esp'
+    tag:
+      - Actors.ACBS
+      - Actors.AIPackages
+      - C.Acoustic
+      - C.Climate
+      - C.Encounter
+      - C.Light
+      - C.Location
+      - C.Music
+      - C.Owner
+      - C.RecordFlags
+      - C.Regions
+      - C.Water
+      - Invent
+  - name: 'Beasts of Tamriel.esp'
+    tag:
+      - Actors.ACBS
+      - C.Encounter
+      - C.Location
+      - C.RecordFlags
+      - C.Regions
+      - C.Water
+      - Invent
+      - Relev
+  - name: 'Solitude Expansion.esp'
+    tag:
+      - C.Location
+      - C.Regions
+  - name: 'JK''s SkyHaven Forge.esp'
+    tag:
+      - C.Regions
+  - name: 'TrueStormsSE.esp'
+    tag:
+      - Sound
+  - name: 'DLCIntegration.esp'
+    tag:
+      - Delev
+      - Invent
+  - name: 'SexLab-AmorousAdventures.esp'
+    tag:
+      - C.Climate
+      - C.Water
+  - name: 'ULRiverwoodForestJKPatch.esp'
+    tag:
+      - C.Water
+  - name: 'LADX_SSE.esp'
+    tag:
+      - Delev
+      - Invent
+      - Relev
+  - name: 'DBM_CRF_Patch.esp'
+    tag:
+      - Factions
+      - Names
+  - name: 'DBM_SkyrimUnderground_Patch.esp'
+    tag:
+      - Invent
+  - name: 'DBM_HelgenReborn_Patch.esp'
+    tag:
+      - Factions
```

Comment:
If you can set whatever editor you use to add a new line/line break at the end of the file, that would be great. Not really a big deal, but it prevents commits "fighting" over changing the last line as some editors add it back and others remove it and just clutters diff views unnecessarily.

Reply:
I took it out on purpose actually since I'm used to dealing with yaml on non-Windows platform,  but I'll try to retain the cr/lf at the end from here on.

Reply:
(Note that the line breaks will be converted by git when cloning the repository, so while it's `CRLF` for you, it's `LF` for me (on Linux) and will be just `CR` for people on Mac OS <X machines. :) Git does the line ending conversion transparently (unless you explicitly tell it not to!).)

Reply:
Good to know.  In my day job I'm a *nix admin but our desktop machines are all MBPs, so I've been insulated from the Windows CRLF mess for the past decade or so. 

On the Windows gaming box I have at home I'm using Notepad++ which should respect the existing format of the file, but I'm thinking I'll just set it to use LF for YAML files to play it safe. That or I could just use Vim in a WSL terminal like a civilized human.  ;)